helix-tui/src/backend/crossterm.rs

```diff
@@ -62,6 +64,7 @@ where
         CrosstermBackend {
             buffer,
             capabilities: Capabilities::from_env_or_default(),
+            supports_keyboard_enhancement_protocol: OnceCell::new(),
```

Comment:
This value is not included in Capabilities because the tty needs to be in raw mode to perform the query and the backend is created before raw mode is enabled